components/service/glean/src/test/java/mozilla/components/service/glean/TestUtil.kt

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean
+
+import org.json.JSONObject
+
+const val PING_SCHEMA_GIT_HASH = "3a15121c"
```

Comment:
We could probably fetch the script using Gradle and pin the hash in the build configuration too, to avoid hardcoding this in the source. Downloading the schema could be done through the `task` described in the related [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1504200#c0)

Reply:
I don't think downloading the schema using gradle is a good idea.  We would run into the same versioning issues we had with glean_parser, but even worse, without a built-in way to manage record, check and manage those versions.

I would be fine with storing the git hash in `sdk_generator.gradle` and injecting that value here, assuming that's possible.

Reply:
> I would be fine with storing the git hash in `sdk_generator.gradle` and injecting that value here, assuming that's possible.

@pocmo any idea if this is possible or not?

Reply:
I've submitted a method that sets the URL to a resource string and then accesses that from the code.  If there's a better method, let me know.